server/build.gradle

```diff
@@ -0,0 +1,34 @@
+apply plugin: 'java'
+apply plugin: 'application'
```

Comment:
Does this build the same binary as the sbt version?

Reply:
Yes, it builds the same binary and creates a shell script. Not identical shell script but it works the same.